pkg/xen-tools/xen-start

```diff
@@ -88,6 +88,13 @@ xl create "$2" -p $EXTRA_ARGS || bail "xl created failed"
 # we may need to wait for domain to come online for us to manipulate it (timing out in under 30 sec)
 ID=$(domID "$1")
 
+if [ $# -eq 3 ]
+then
+  PRODUCT_NAME="OpenStack Compute"
```

Comment:
This doesn't make any sense to me. We can not provide all the meta-data which openstack provides since it plainly does not exist and this creates the expectation that we can and will provide that.